service/src/main/java/bio/terra/pipelines/common/utils/PipelineVariableTypesEnum.java

```diff
@@ -92,6 +98,10 @@ public String validate(PipelineInputDefinition pipelineInputDefinition, Object v
       if (stringCastValue == null || !(stringCastValue.endsWith(fileSuffix))) {
         return "%s must be a path to a file ending in %s".formatted(fieldName, fileSuffix);
       }
+      if (!VALID_STRING_PATTERN.matcher(stringCastValue).matches()) {
+        return "%s must be a file path containing only alphanumeric characters or the following symbols: - _ . = /"
```

Comment:
i think you can use the same error string here as in the string error message (in this case it's not really relevant if its a file path) just to help with refactoring some of these strings out